hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java

```diff
@@ -64,6 +64,8 @@
    */
   public static final String CHORE_POOL_SIZE = "hbase.cleaner.scan.dir.concurrent.size";
   static final String DEFAULT_CHORE_POOL_SIZE = "0.25";
+  public static final String LOG_CLEANER_CHORE_SIZE = "hbase.log.cleaner.scan.dir.concurrent.size";
```

Comment:
So now we will have (by default) a pool of 1 thread for scan and cleaning of Archived HFiles.
For old WALs the thread pool's default size will be 25% of #cores 
Can we add some code level comments here?

Reply:
I think, pool of 1 thread (by default) is for scanning and cleaning the old WALs. And pool with 25% of #cores (by default) is for archived store files.
+1 adding code level comments.